Error Report: 2025-05-28T14:17:37.835Z Erreur dans main():

An error occurred:

```
Error: Cannot find module 'bluebird'
Require stack:
- D:\tri\test\node_modules\hexo\dist\hexo\index.js
- D:\tri\test\index.js
```

--- a/package.json
+++ b/package.json
@@ -1,6 +1,6 @@
 {
   "name": "adminpanel",
-  "version": "3.0.2",
+  "version": "3.0.3",
   "main": "index.js",
   "scripts": {
     "build": "hexo generate",

